internal/pkg/model/manifest.go

```diff
@@ -21,11 +21,11 @@ type RecordPaths interface {
 	// GetRelativePath - for example path of the object inside parent object/path.
 	GetRelativePath() string
 	// SetRelativePath - for example path of the object inside parent object/path.
-	SetRelativePath(string)
+	SetRelativePath(p string)
 	// GetParentPath - for example path of the parent object.
 	GetParentPath() string
```

Comment:
I don't see any benefit in it at all.

Reply:
If you call the names properly you know what it accepts (`string` or `int`) is quite generic. In this case it is `p` which means path. It could also prevent some naming issue of method in interface against it's parameters. But that's my opinion